tiled/adapters/tiff.py

```diff
@@ -216,8 +216,9 @@ def _safe_asarray(seq, index=None):
     https://github.com/cgohlke/tifffile/commit/18315846ebf0b31126dfdeffee8e0ef30d81b31c#diff-d8d5cccae6533b861bdf6c0c20c6f49415978effec9a13797909535f3fc7390dL9563-R9746  # noqa
     """
 
-    try:
-        return seq.asarray(index=index)
-    except TypeError:
-        # usage for versions before 2021.10.10
-        return seq.asarray(file=index)
+    return seq.asarray()[index]
```

Comment:
This reads the entire sequence (i.e. all the TIFF files) and then slices out the part we want. The existing code reads only exactly the data we need, which is of course faster. I am sure that tifffile still supports this somehow, but it's not immediately clear how. Let's address this tifffile change in a separate PR.

Reply:
Fixed in https://github.com/bluesky/tiled/pull/206